src/utilities/schemaHandlers.js

```diff
@@ -7,6 +7,40 @@ export const AJV_OPTIONS = {
   }
 }
 
+const processAnyOf = (prop, allowedTypes) => {
+  /*
+  Handles AnyOfs in schema.
+  Removes unneccessary dropdowns for json schema primitive types.
+  Otherwise, adds default titles to dropdown of allowed types/subschemas
+  */
+  const isAllowedType = prop.anyOf.some(option => allowedTypes.includes(option.type))
+  const isOptional = prop.anyOf.some(option => option.type === 'null')
+  const isDecimal = prop.anyOf.some(option => option.type === 'number') && prop.anyOf.some(option => option.type === 'string')
+  // Replace dropdown with input box
+  if (prop.anyOf.length <= 3 && isDecimal) {
+    prop.type = 'number'
```

Comment:
nit: consider making this a ternary or if/else statement to avoid setting twice.